Oct week3 (#180)
Open-Attestation/documentation-website: website/docs/integrator-section/verifiable-document/did/revoking-document.md

- Some kind of DID documents:
  1. Base DID document (`ISSUER_IDENTITY`: `identityProof.identifier` against `ethr` DID)
  2. DID-DNS document (`ISSUER_IDENTITY`: `identityProof.identifier` against a [DNS-TXT](/docs/docs-section/how-does-it-work/issuance-identity))

> - A deployed `documentStore`, click [here](/docs/integrator-section/verifiable-document/ethereum/document-store) for the steps.

Reviewer: this is linking to the `ethereum` section. Since we have a DID section for deploying the document store should we link to that instead?

Author: Updated to the DID tutorial

<!-- TBD v3 document sample when that releases -->

> - The `<DEPLOYED_DOCUMENT_STORE_LOCATION>` is your `documentStore` location. It must be **deployed** on the Ethereum block chain (for now. Further implementations will relax this requirement, so that DID documents will truly be gasless).

Reviewer: As we have implemented the OCSP alternative, we can remove the line in the brackets. Also, `block chain` should be one word. 

```suggestion
- The `<DEPLOYED_DOCUMENT_STORE_LOCATION>` is your `documentStore` location. It must be **deployed** on the Ethereum blockchain. 
```

Author: added your suggestion

open-attestation document-store revoke --address <DOCUMENT_STORE_LOCATION> --hash <HASH_OF_DOC(S)>  --network <NETWORK> --encrypted-wallet-path <PATH_OF_WALLET>
```

> >**Note:** 

Reviewer: There is another line above this `**Note:** Before running the command, replace the variables with the actual names to fit your case.`

<img width="862" alt="Screenshot 2023-10-27 at 3 19 25 PM" src="https://github.com/Open-Attestation/documentation-website/assets/31716292/6fb88256-4e2e-4e74-a491-941b91b9c376">

Looks weird having 2 notes so close to each other, also I think the note above can be removed. In other sections I see that we either explain what each parameter is e.g. Under DID -> Signing Documents we say `In the example above: ...` 

And under DID -> Wrapping Documents we say `Replace the folder/file names below and run the command:` 

Can we follow one of these 2 and remove this note? We should stick to 1 or at most 2 phrases for these instructions. My personal opinion is that the way we do it in DID -> Signing Documents is the best.

Author: Removed the first note and updated the description for consistency
